prow/plugins/override/override.go

```diff
@@ -253,18 +253,14 @@ func authorizedGitHubTeamMember(gc githubClient, log *logrus.Entry, teamSlugs ma
 	slugs := teamSlugs[fmt.Sprintf("%s/%s", org, repo)]
 	slugs = append(slugs, teamSlugs[org]...)
 	for _, slug := range slugs {
-		for _, team := range teams {
-			if team.Slug == slug {
-				members, err := gc.ListTeamMembers(org, team.ID, github.RoleAll)
-				if err != nil {
-					log.WithError(err).Warnf("cannot find members of team %s in org %s", slug, org)
-					continue
-				}
-				for _, member := range members {
-					if member.Login == user {
-						return true
-					}
-				}
+		members, err := gc.ListTeamMembersBySlug(org, slug, github.RoleAll)
+		if err != nil {
+			log.WithError(err).Warnf("cannot find members of team %s in org %s", slug, org)
+			continue
+		}
+		for _, member := range members {
+			if member.Login == user {
```

Comment:
Unrelated to your change, but this kind of comparison needs to be done case-insensitively,  https://github.com/BenTheElder is the same as https://github.com/bentheelder